### PR TITLE
fix: add namespace to check push permissions ref

### PR DESCRIPTION
**Pull request by jpower432**
# Description

This PR changes the image reference passed to `oc-mirror` to add the user-defined namespace.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Manual testing was done here to verify
- [x] Add unit test

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by afflom** [comment]
testing now

File: pkg/cli/mirror/mirror.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ func (o *MirrorOptions) Complete(args []string) error {
 	return nil
 }
 
-func (o *MirrorOptions) Validate() error {
+func (o *MirrorOptions) Validate(args []string) error {
```
Comment (estroz) [inline pkg/cli/mirror/mirror.go:106]: Can you add a unit test for this

Comment (jpower432) [reply pkg/cli/mirror/mirror.go:106]: Will do

Comment (estroz) [reply pkg/cli/mirror/mirror.go:106]: Also #224 has some changes that improve validation in `Complete()` relevant here.

**Comment by afflom** [comment]
I just tested this with registry2 and verified that it works with a user configured namespace.

**Pull request merged**